### PR TITLE
Mono repo

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: app/pubspec.lock [hunk 1]
```diff
@@ -674,4 +674,4 @@ packages:
     source: hosted
     version: "2.1.15"
 sdks:
-  dart: ">=2.1.0-dev.5.0 <3.0.0"
+  dart: ">=2.1.0 <3.0.0"
```
Comment (isoos) [inline app/pubspec.lock]: Let's just do `>=2.1.0-dev.9.4` (consistent with Dockerfile)

File: app/pubspec.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: pub_dartlang_org
 author: Dart Team <misc@dartlang.org>
 description: The pub.dartlang.org website.
 environment:
-  sdk: ">=2.0.0 <3.0.0"
+  sdk: ">=2.1.0 <3.0.0"
```
Comment (isoos) [inline app/pubspec.yaml]: `>=2.1.0-dev.9.4`

File: pkg/_popularity/mono_pkg.yaml [hunk 3]
```diff
@@ -0,0 +1 @@
+# See https://github.com/dart-lang/mono_repo for details
```
Comment (isoos) [inline pkg/_popularity/mono_pkg.yaml]: Do we need these empty files around?

Comment (kevmoo) [reply pkg/_popularity/mono_pkg.yaml]: They allow `mono_repo --recursive pub get` to work...

**Comment by kevmoo** [comment]
@isoos – PTAL

**Pull request merged**